rspec-mode.el

```diff
@@ -781,12 +791,18 @@ or a cons (FILE . LINE), to run one example."
       (rvm-activate-corresponding-ruby))
 
   (let ((default-directory (or (rspec-project-root) default-directory)))
-    (compile (rspec--vagrant-wrapper
-              (rspec--docker-wrapper
-               (mapconcat 'identity `(,(rspec-runner)
-                                     ,(rspec-runner-options opts)
-                                     ,target) " ")))
-             'rspec-compilation-mode)))
+    (compile
+     (rspec-compile-command target opts)
+     'rspec-compilation-mode)))
+
+(defun rspec-compile-command (target &optional opts)
+  "Composes RSpec command line for the compile function"
+  (let ((default-directory (or (rspec-project-root) default-directory)))
```

Comment:
We can do. This is my first PR on elisp, so I tried to not move/refactor a lot of things.

Do you have ideas about how to refactor this? extract to another function?